osu.Game/Overlays/Changelog/UpdateStreamBadge.cs

```diff
@@ -112,21 +103,23 @@ private void updateState()
             // Expand based on the local state
             bool shouldExpand = Active.Value || IsHovered;
 
+            bool allHighlighted = SelectedTab.Value == null && !externalDimRequested;
+
             // Expand based on whether no build is selected and the badge area is hovered
-            shouldExpand |= SelectedTab.Value == null && !externalDimRequested;
+            shouldExpand |= allHighlighted;
 
             if (shouldExpand)
```

Comment:
i would probably just make a separate if section for `allHighlighted` rather than try and squeeze them in existing cases.